docs/source/python/csv.rst

```diff
@@ -64,6 +65,16 @@ with the file path you want to read from::
    3       23.68  3.31    Male     No  Sun  Dinner     2
    4       24.59  3.61  Female     No  Sun  Dinner     4
 
+To write CSV files, just call :func:`write_csv` with a
```

Comment:
it might be worth a note that not all arrow types are supported yet.

Reply:
Added a note below (and in the C++ docs too).